services/diversity/src/services/richness.js

```diff
@@ -110,6 +110,48 @@ module.exports = () => {
           return data;
       }
     },
+
+    /**
+     * Get values for richness species gaps in the given area
+     *
+     * @param {String | Number} areaType area type
```

Comment:
We keep making this mistake, areaType is always string, is areaId the one that can be number or string.

Also please fix the doc in the older two functions

Reply:
Fixed